.evergreen/config_generator/components/funcs/install_uv.py

```diff
@@ -0,0 +1,17 @@
+from config_generator.etc.function import Function
+from config_generator.etc.utils import bash_exec
+
+from shrub.v3.evg_command import EvgCommandType
+
+
+class InstallUV(Function):
+    name = 'install-uv'
+    commands = bash_exec(
+        command_type=EvgCommandType.SETUP,
+        working_dir='mongo-cxx-driver',
+        script='.evergreen/scripts/uv-installer.sh'
+    )
```

Comment:
Can this be modified to install `uv` on a cache/scratch space specific to `mongo-cxx-driver` and the desired `uv` version?

Reply:
Added a new `set-cache-dir` EVG function which defines a `MONGO_CXX_DRIVER_CACHE_DIR` EVG expansion. This expansion points to a dedicated cache directory for the C++ Driver (e.g. `~/.cache/mongo-cxx-driver` or equivalent). `uv` is now installed into this cache directory in the `uv-0.5.9` (versioned) subdirectory and also defines a `UV_INSTALL_DIR` EVG expansion for subsequent use in EVG scripts.